rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/TclLauncher.xtend

```diff
@@ -0,0 +1,72 @@
+package org.testeditor.rcp4.tcltestrun
+
+import java.io.File
+import org.eclipse.core.resources.IProject
+import org.eclipse.core.resources.ResourcesPlugin
+import org.eclipse.jdt.junit.JUnitCore
+import org.eclipse.jface.viewers.IStructuredSelection
+import org.gradle.tooling.GradleConnectionException
+import org.gradle.tooling.GradleConnector
+import org.gradle.tooling.ProjectConnection
+import org.gradle.tooling.ResultHandler
+import org.gradle.tooling.events.ProgressEvent
+import org.gradle.tooling.events.ProgressListener
+import org.slf4j.LoggerFactory
+import org.testeditor.tcl.dsl.ui.testlaunch.Launcher
+
+class TclLauncher implements Launcher {
+	static val logger = LoggerFactory.getLogger(TclLauncher)
+
+	def void showResult(String path) {
+		val testResult = new File(path)
+		var count = 0
+		val maxcount = 20 // wait max 2s
+		while (!testResult.exists && count < maxcount) {
```

Comment:
Why do we need to wait here? I would assume that the callback for `onComplete` is triggered when the file is already there. If not, we should put a comment here explaining that.